src/main/java/com/compatibility/Compatibility.java

```diff
@@ -0,0 +1,35 @@
+package com.compatibility;
+
+import com.compatibility.tinkers.ToolBrokenCheck;
+import net.minecraft.item.ItemStack;
+import org.jetbrains.annotations.Nullable;
+
+/**
+  * This class is to store the methods that call the methods
+  * to check for miscellaneous compatibility problems
+  */
+public class Compatibility
+{
+    
+    /**
+      * This method checks to see if STACK is able to mine anything.
+      * It goes through all compatibility checks.
+      *
+      * @param stack the item in question.
+      * @param tool the name of the tool.
+      * @return boolean whether the stack can mine or not.
+      */
+    public static boolean checkMiningCompatibility(@Nullable ItemStack stack, @Nullable String tool)
+    {
+        if (ToolBrokenCheck.checkTinkersBroken(stack, tool))
+        {
+            return true;
+        }
+        return false;
+    }
+
+    public Compatibility()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Hide this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1118)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1213)